zng/resolver/marshal.go

```diff
@@ -138,10 +144,10 @@ func typeFull(name, path string) string {
 type TypeStyle int
 
 const (
-	TypeStyleNone = iota
-	TypeStyleSimple
-	TypeStylePackage
-	TypeStyleFull
+	StyleNone = iota
```

Comment:
Make these TypeStyles rather than just ints.
```suggestion
	StyleNone TypeStyle = iota
```